agent/log_monitor.py

```
@@ -186,6 +195,21 @@ def _sig(logger_name: str, message: str) -> str:
+    """Best-effort: hand an operational failure to the incident tracker.
+
+    Import is local and the whole call is guarded because this runs inside a
+    ``logging.Handler`` — a failure here must never propagate into the emit
+    path that every other log line in the process shares.
+    """
+    try:
+        from agent.operational_incidents import record_operational_error
+
+        record_operational_error(logger_name, message)
+    except Exception:  # nosec B110 - incident tracking is best-effort
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)